### PR TITLE
Remove noop --timestamp param from `parsec core run` CLI command

### LINKED ISSUE #3345: `parsec core run` has a useless `--mountpount` parameter

### PR DESCRIPTION
**Pull request by touilleMan**
closes #3345

### CODE REVIEW

**Review by FirelightFlagboy — COMMENTED** [review]
I don't see how the issue #3345 is related to this PR

The issue point a problem with the option `--mountpoint`
but here we remove the option `--timestamp`

Maybe change the title of the issue if it was miss labeled, to prevent confusion in the changelog :thinking:

**Comment by touilleMan** [comment]
> The issue point a problem with the option --mountpoint
> but here we remove the option --timestamp

The issue was precisely the `--timestamp` did nothing, so there is two way to fix this 1) implement it 2) remove it
We don't have need for this feature for the moment, so the fix is 2)

**Comment by FirelightFlagboy** [comment]
> The issue was precisely the `--timestamp`

Again, it's not the cause

The issue's title is

> `parsec core run` has a useless **`--mountpount`** parameter

**Comment by touilleMan** [comment]
Ha ! I finally understand: I wrote `--mountpoint` by mistake in the original issue ! I've corrected it ;-)

File: newsfragments/3345.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove noop --timestamp param from `parsec core run` CLI command
```
Comment (FirelightFlagboy) [inline newsfragments/3345.misc.rst:1]:
```suggestion
Remove noop `--timestamp` param from `parsec core run` CLI command
```

**Review by FirelightFlagboy — APPROVED** [review]
Just a minor suggestion for the fragment, but it's ok

File: newsfragments/3345.misc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Remove noop `--timestamp`` param from `parsec core run` CLI command
```
Comment (FirelightFlagboy) [inline newsfragments/3345.misc.rst:1]:
```suggestion
Remove noop `--timestamp` param from `parsec core run` CLI command
```

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

**Pull request merged**